rustsec/Cargo.toml

```diff
@@ -24,8 +24,10 @@ toml = "0.7"
 url = { version = "2", features = ["serde"] }
 
 # optional dependencies
-cargo-edit = { version = "0.9", package = "cargo-edit-9", optional = true, default-features = false,  features = ["upgrade"] }
-crates-index = { version = "0.19", optional = true }
+cargo-edit = { version = "0.9", package = "cargo-edit-9", optional = true, default-features = false, features = [
+    "upgrade",
+] }
+tame-index = { version = "0.3", features = ["git", "sparse"], optional = true }
```

Comment:
I just wanted to say I appreciate `git` and `sparse` being separate features that can be toggled individually :heart: